src/adapters/poi/bragi_poi.js

```diff
@@ -48,19 +38,29 @@ export default class BragiPoi extends Poi {
         name = label;
       } else {
         name = label.slice(0, splitPosition);
-        alternativeName = label.slice(splitPosition + 1);
       }
     }
     }
 
-    super(id, name, alternativeName, type, {
+    super(id, name, type, {
       lat: feature.geometry.coordinates[1],
       lng: feature.geometry.coordinates[0],
     }, poiClassText, poiSubclassText, geocodingProps.bbox);
 
     this.value = label;
-
     this.queryContext = queryContext;
+    this.address = {
+      name: geocodingProps.address?.name || geocodingProps?.name,
```

Comment:
What is an address "name", and what is the difference with a "label"?
This change is the occasion to use more explicitly named properties, maybe we can find something better?

Reply:
Ok let's find a proper name, for now we have:
name: "X Rue Lacépède"
city: Paris
country: France

And the function `getFullAddress` in `pois.js` returns the string "X Rue Lacépède, Paris, France".

We can try to format `X Ru lacépède` from street info, but the order might depends on localization.

Reply:
differences between a "label" and "getFullAddress" may not be explicit.

Reply:
Sorry, I'm still not sure what is this address 'name'…
Looking at exemples it looks like it's actually the street part of the address (number + street name).
If that's the case, it probably:
 - should be renamed to something more explicit, like simplify `street` or smth,
 - should be allowed to be empty, for exemple for cities and such,
 - should be distinguished from the name of the POI, when it exists (in idunn_poi the POI name is used as a fallback for it, it may induce confusion.)

Reply:
I remove the name field. We now have :
`street`, `city`, `country`, `label` (used by PoiPanel and PoiPopup)